rpa-bot/1.3/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -28,12 +30,28 @@ include::rpa-home::partial$windows-versions-support.adoc[]
 
 |===
 
-=== Requirements for Secure Session Technology
+=== System Settings
+
+* Display resolution: 
+** Minimum: 1280 x 1080
+** Recommended: Full HD 1080p (1920 × 1080) or higher
+* Scaling of text and other items on the screen: 100 %
+* _Clear Type_: turned off
+
+=== Requirements for the Secure Session Technology
+
+Secure Sessions are hidden Desktop sessions. They run in the background and only the bot can access them. An additional advantage of using Secure Sessions is, that the bot can multiply itsself to run several process instances in parallel.
+
+To use Secure Sessions you must ensure that all prerequisites are met and all responsible persons are informed.
```

Comment:
Any way to make this active voice?

Reply:
I wonder if you can make this lowercase in this case? It's kind of like:
connected apps, the Connected Apps feature
Lowercase the common noun and uppercase the proper noun.

Reply:
The problem is that it is not defined who does this and in what way.

Reply:
I think we only use it as proper noun.
There is a patent and I think the name is registered.

Reply:
Suggest reviewing all mentions and making sure they're correct.

Reply:
I upper-cased all occurrences of Secure Session and Desktop Session, because this is Servicetrace terminology.
see https://techcommunity.microsoft.com/t5/ask-the-performance-team/sessions-desktops-and-windows-stations/ba-p/372473 for general information about 'Sessions'
Desktop Session = "consists of all of the processes and other system objects that represent a single user’s logon session." and the user is currently logged in. 
Secure Session = same but hidden. Because it is a Remote Desktop Session at the local loopback address.
@Cristian-Venticinque maybe we can add a user-friendly definition of these two terms to the Glossary